### DISCUSSION TITLE
Sharpie set

### DISCUSSION THREAD

**Discussion by csanii123** [Sakati84/3DPrintColorizer#23] [General]
My wife bought this 34 pack years ago at Costco...not sure why since as you can see almost all are still in the package. I pulled out 4 markers and used them for my test print and they worked pretty well. 
I don't see the set at Costco.ca anymore but I do see a 24 pack at Staples.ca for $25
So if you want lots of colors...
![Sharpies](https://user-images.githubusercontent.com/67832530/128610783-cf34c70d-d5de-47de-966c-935bdab0afaa.jpg)

**Comment by Sakati84** [comment]
In Germany you can get a 28 pack for 16€. They are also reduced quite often. I once got them for 25cent each. Can you also share pictures of your test print?

**Reply by csanii123** [reply]
25 cents is a very good price. Below is my test print. Nothing special. Just a cube, cylinder and tube combined and each with different colors (4 colors + transparent). I stopped the print half way through as this was just a test to see if I had my E3 v2 clip-on pen holder setup right. I mounted the penholder in front of the extruder as that was easiest but I realized the pen rack isn't long enough so I think I will rework my penholder so it is on the side instead that way I can use the standard pen rack.
![MultiColorTestPrint2](https://user-images.githubusercontent.com/67832530/128614549-3d5d0910-2308-4315-84d7-d3325705eadc.jpg)
